config/test.json

```diff
@@ -6,12 +6,9 @@
 	"public": "../public/",
 	"services": {
 		"calendar": "https://schul.tech:3000",
-		"content": "https://content.schul-cloud.org",
-		"notification": "https://hpi-schul-cloud.org:3030"
```

Comment:
very strange, here the notification service points to the the server, on port 3030, but should have been 3031?

Reply:
Yes and even the code in the old client is strange.
It's lock sometimes that the server where an proxy to the node-notification-service.